wiki/Mascots/en.md

```diff
@@ -64,36 +64,36 @@ Agent J was also featured in LuigiHann's skin, [Elite Beat osu! HD (1.0 Complete
 
 > Don is the protagonist of the [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin) series. He is a taiko drum with a white edge and four legs, a red face (which appears to be moly orange), and a light blue body. Don's dream is to share the beauty of Taiko with the world. Three years have passed since he moved into the Wada House, and he's become quite popular around town. He has a monstrous appetite and sometimes has expensive shopping at the Wada House which may take over for the worst. He usually ends his sentences with "Ta-don", meaning "Ba-dum" in Japanese.
```

Comment:
If it was a quote taken from somewhere, leave as is.
`he's become` -> `he has become` (diff not showing)
```suggestion
> Don is the protagonist of the [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin) series. He is a taiko drum with a white edge and four legs, a red face (which appears to be moly orange), and a light blue body. Don's dream is to share the beauty of Taiko with the world. Three years have passed since he moved into the Wada House, and he has become quite popular around town. He has a monstrous appetite and sometimes has expensive shopping at the Wada House which may take over for the worst. He usually ends his sentences with "Ta-don", meaning "Ba-dum" in Japanese.
```

Reply:
It looks like this *might* have been mostly copied from [here](https://gamicus.gamepedia.com/Characters_of_Taiko_no_Tatsujin#WadaDon) ~~... or maybe they copied from us?~~ ¯\\\_(ツ)\_/¯

but since it looks like it was copied from there, I'll include some kind of link to that page.